cypress/integration/pages/onDemandRadio/index.js

```diff
@@ -11,7 +11,13 @@ const pageType = 'onDemandRadio';
 Object.keys(config)
   .filter(service => serviceHasPageType(service, pageType))
   .forEach(serviceId => {
-    const { variant, name: service } = config[serviceId];
+    // eslint-disable-next-line prefer-const
+    let { variant, name: service } = config[serviceId];
```

Comment:
Am I right in thinking this reconstruction of the service is for accessing the services within `cypress/support/config/settings.js` when doing something like `config[service].name`?

Taking Zhongwen as an example - `config[service].name` accesses the config for `zhongwenTrad` and returns the name, "zhongwen" - which is the same as the `service` prop before you capitalise it and add the variant.

So for example, in `cypress/integration/pages/onDemandRadio/tests.js` does: 

```js
const language = appConfig[config[service].name][variant].lang;

// where service = 'zhongwenTrad'
```

have to access the config via `config[service].name` (which needs the service to be service + capitalised variant)

or could it just be:

```js
const language = appConfig[service][variant].lang;

// where service = 'zhongwen'
```

Basically - is there any instance where the `service` prop received in the test index does not equal `config[service].name`?

Reply:
changed